plugins/datocms-plugin/TS-README.md

```diff
@@ -28,7 +28,7 @@ You can find your API token under the Settings > API tokens section of your proj
 
 1. **Create your DatoCMS project**
 
-[![Clone DatoCMS project](https://dashboard.datocms.com/clone/button.svg)](https://dashboard.datocms.com/clone?projectId=60908&name=Homepage+Starter)
+[![Clone DatoCMS project](https://dashboard.datocms.com/clone/button.svg)](https://dashboard.datocms.com/clone?projectId=60908\&name=Homepage+Starter)
```

Comment:
Hmmm I'm a little suspect of this additional `\` inserted here.

Reply:
Yeah I also was, and forgot to add a comment earlier 🤦 – I looked at the rendered file and I think we're fine (link seems to work – but please check me, having a headache all day :/)): https://github.com/gatsbyjs/homepage-starters/blob/ed2380c4228e4e48470a7c6ad743d4260ed2916a/plugins/datocms-plugin/TS-README.md#quick-start

I didn't bother at trying to track down where/when things changed, but lemme reinstall packages and run `yarn create-readmes` again, maybe it (escaping the ampersand) will go away again.

Also found this: https://github.com/syntax-tree/mdast-util-to-markdown/issues/53

Reply:
> I didn't bother at trying to track down where/when things changed, but lemme reinstall packages and run `yarn create-readmes` again, maybe it (escaping the ampersand) will go away again.

No changes :v: